perl/p5-future/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.branches      5.26
+perl5.setup         Future 0.38 ../by-authors/id/P/PE/PEVANS
+
+platforms           darwin
+maintainers         {@chrstphrchvz gmx.us:chrischavez} openmaintainer
+license             {Artistic-1 GPL}
+
+supported_archs     noarch
+
+description         Future - represent an operation awaiting completion
+
+long_description    ${description}
```

Comment:
I generally think if your long_description is the same as your description something is wrong. That said, I'm not going to block this, but please don't do that going forward. IMHO, `long_description` should give you enough information to figure out why you might want to install something. The reason I'm setting it aside here is that this is a library and probably no one actually wants it on its own.

Reply:
Generally, I think trying is better than not trying.